### DISCUSSION TITLE
Pass custom Headers to endpoint created using svix

### DISCUSSION THREAD

**Discussion by PurvaSatish** [svix/svix-webhooks#1494] [Q&A]
I am using Svix 1.23.0 version, I need to add a header along with payload to send webhooks, Is there a way to send custom header along with webhooks here. 
svix.message.create(
        applicationId, MessageIn(
            eventType = eventType, payload = convertToMap(orderPayload), eventId = eventId
        ),
        options = PostOptions().idempotencyKey(eventId)
    )
    I need to send auth token generated by us not using svix while sending webhooks

**Comment by svix-jplatte** [comment]
You can send additional headers by adding them to the endpoint before sending the message (see https://api.svix.com/docs#tag/Endpoint/operation/v1.endpoint.update-headers).

If you want to send different values per messages, that's not currently supported in the OSS version - if I understood the question correctly you are self-hosting the open source Svix server, right?
